app/markets/ActivityTable.tsx

```diff
@@ -55,6 +61,29 @@ export const ActivityTable = ({ id }: { id: string }) => {
       }),
   });
 
+  const { data: aiAgentsList } = useQuery({
+    queryKey: ['getAIAgents'],
+    queryFn: async () => {
+      const options: LatestResultArgs = {
+        queryId: 3582994,
```

Comment:
Out of curiosity: how do we get this number?

Reply:
the answer to that should be on a constant variable so it's more clear where the number comes from:

`const DUNE_QUERY_ID = 3582994`

Reply:
Improved

Reply:
> Out of curiosity: how do we get this number?

It's the agents info query id on dune